src/Components/Notes/NoteCard.jsx

```diff
@@ -214,15 +248,26 @@ const ShowMore = ({onClick, expandText}) => {
 
 
 const CardActions = ({
```

Comment:
Please use https://mui.com/material-ui/api/card-actions/ if possible

Reply:
This card actions method was made by you.